doc/tests/manual/protocol.md

```diff
@@ -1,359 +1,104 @@
 # Manual Test Protocol
 
```

Comment:
Start explaining that we use Gherkin syntax here, so that tests can be automated more easily later. And that a template is at the very end.